PlexusBuffIcons.lua

```diff
@@ -88,7 +88,7 @@ local options = {
             order = 51, width = "single",
             type = "toggle",
             name = L["Only Mine"],
-            disabled = function(info) return not PlexusBuffIcons.db.profile.showbuff end, --luacheck: ignore 212
```

Comment:
why remove this? the check still fails.

Reply:
I didn't remove that. I don't know why it happens like that